docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst

```diff
@@ -0,0 +1,440 @@
+
+.. image:: stomach_diagram.svg
+
+Stomach cancers tend to develop slowly over many years. Before a true cancer develops, pre-cancerous changes often occur in the inner lining (mucosa) of the stomach. These early changes rarely cause symptoms and therefore often go undetected. Most (about 90% to 95%) cancers of the stomach are adenocarcinomas. A stomach cancer or gastric cancer almost always is an adenocarcinoma. These cancers develop from the cells that form the innermost lining of the stomach (the mucosa). Correa pointed out that the human gastric carcinogenesis is a slow progressive, multistep, and multifactorial pathology process. The multistep process is composed of chronic superficial gastritis, atrophy gastritis, intestinal metaplasia (IM), dysplasia (DYS), and adenocarcinoma (malignant neoplasm).
+
+ -  Note that there are cardia (50-60%) and non-cardia cancers (40-50%) both classified by GBD as stomach cancers
+ -  Cardia cancers are not associated with H. pylori infection while non-cardia cancers are. There is evidence written that says cardia cancers may be protected by H. pylori. Incidence is cardia cancers   are rising while stomach cancer overall and non-cardia cancers are decreasing (better hygiene?) 
+ - This pools all stomach cancer
+
+
+.. image:: correa_cascade.svg
+
+.. image:: stomachcancer_lifehistory.svg
+
+.. _1.1:
+
+1.1 Project overview
+--------------------
+
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+
+To estimate the yearly number of cases of stomach cancer detected under specific screening practices and the yearly number of deaths from undetected stomach cancer (both in unit of per 100,000 insured person-years) in order to identify pay-out trends for critical insurance claims (CII).  
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal diagram
+------------------
+
+**Outcome (O)**:
+
+  - stomach cancer 
+
+**Exposure (E)**:
+  
+  - h.pylori
+
+
+
+.. _3.2:
+
+3.2 Effect and impact size
+--------------------------
+
+RR of H. pylori = 4.5 
+Exposure of H. pylori = 44%
+PAF of H. pylori = 60%
+
+.. _4.0:
+
+4.0 Intervention
+++++++++++++++++
+
+Scale-up of stomach cancer screening coverage among insured population 
+
+.. _4.1:
+
+4.1 Simulation scenarios
+------------------------
+
+:underline:`Baseline scenario`
+
+  * no screening coverage
+
+:underline:`Alternative scenario`
+
+30% of insured Chinese male/female initiated stomach cancer screening in 2020, stay 30% for one year then linearly project to 75% by 2030 and hold constant till 2040 for blended provinces, where:
+
```

Comment:
Can be helpful to note for the SE's that these are stand-in values so they are aware these screening coverage numbers may change after initially implemented (esp. since we haven't confirmed these numbers with SR yet).